src/lib.rs

```diff
@@ -221,12 +221,11 @@ pub mod prelude {
         Database as MongoDatabase,
     };
     #[doc(no_inline)]
-    pub use crate::ToRepository;
-    #[doc(no_inline)]
     pub use crate::{
-        f, field, mongo::Cursor, operator::*, pipeline, sync_indexes, BulkUpdate, BulkUpdateResult,
+        f, field, operator::*, pipeline, sync_indexes, BulkUpdate, BulkUpdateResult,
         BulkUpdateUpsertResult, CollectionConfig, Index, IndexOption, Indexes, Model, Repository,
         SortOrder,
+        repository::MongodmCollectionExt, ToRepository,
```

Comment:
```suggestion
        repository::MongodmCollectionExt as _, ToRepository,
```
We probably don't need this trait to be nameable most of the time since it's an extension trait

Reply:
Same could be said for the `ToRepository` trait as part of the prelude though.

However, I vaguely remember that having multiple "as _" exports imported in a project really confused IntelliJ's Rust Language Server. Not sure if it's an issue anymore though

Reply:
Oh yeah, that was the reason. Could you give it a try?

Reply:
Seems fine now